app/models/task.py

```diff
@@ -1,5 +1,47 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from sqlalchemy import ForeignKey
 from ..db import db
+from typing import Optional
+from datetime import datetime
+from typing import Optional
 
 class Task(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    title: Mapped[str]
```

Comment:
Like my comment above in `goal.py` consider which columns for this model should be nullable and which should not be. Like `title` for `Goal`, I also would argue that `title` for `Task` should also not be nullable.